TACC/Core-CMS
# feat: CMD-97 add content to news page

> [!NOTE]
> Abandoned since 0e301198 (whose merge conflicts were incorrectly resolved) because this (as of f1dc83cc) seems to have been half done in `main` already.

## Overview

Allow CMS editor to add content to (top of) news page.

> [!TIP]
> Technically, we _could_ do this via [TACC/Core-CMS-Plugin-Remote-Content](https://github.com/TACC/Core-CMS-Plugin-Remote-Content) since #868, **but** page might load slower.

## Related

- [CMD-97](https://tacc-main.atlassian.net/browse/CMD-97)
- similar to #868
- alternative to #850

## Changes

- **added** placeholder `fixed_content`
- **changed** blog content to render in `fixed_content`

## Testing

0. Have a Blog installed.
1. <details><summary>Verify <strong>main Blog list page</strong> can have unique content above Blog content.</summary>

    1. Load main Blog list page ([example](https://tacc.utexas.edu/news/latest-news/)) as editor.
    2. Add content to main Blog list page.
    3. Verify content appears above the Blog content.
    4. Visit other Blog pages:
        - an article page ([example](https://tacc.utexas.edu/news/latest-news/2024/09/11/escape-from-hurricanes-with-driverless-cars/))
        - a list page for a category ([example](https://tacc.utexas.edu/news/latest-news/category/press-release/))
        - a list page for a tag ([example](https://tacc.utexas.edu/news/latest-news/tag/frontera/))
    5. Verify new content does **not** appear.
    
    </details>
2. <details><summary>Verify other blog pages can <strong>not</strong> have unique content above Blog article.</summary>

    1. Load other Blog pages as editor:
        - an article page ([example](https://tacc.utexas.edu/news/latest-news/2024/09/11/escape-from-hurricanes-with-driverless-cars/))
        - a list page for a category ([example](https://tacc.utexas.edu/news/latest-news/category/press-release/))
        - a list page for a tag ([example](https://tacc.utexas.edu/news/latest-news/tag/frontera/))
    2. Try to add content to page.
    3. Verify content can **not** be added above Blog content.
    
    </details>

## UI

…
